components/suggest/src/db.rs

```diff
@@ -139,12 +140,15 @@ impl<'a> SuggestDao<'a> {
                      SELECT s.id, k.rank, s.title, s.url, s.provider, k.confidence, k.keyword_suffix
                      FROM suggestions s
                      JOIN pocket_keywords k ON k.suggestion_id = s.id
-                     WHERE k.keyword_prefix = :keyword_prefix",
+                     WHERE k.keyword_prefix = :keyword_prefix
+                     ORDER BY s.provider
+                     LIMIT :suggestions_limit",
```

Comment:
@linabutler You mentioned that I don't need to use a binding parameter for this, but I feel like it's a bit more readable than passing into `providers_to_sql_list`. Is there a reason to not using binding params?

Reply:
Oops, I left a review comment earlier, and didn't see your reply; the binding param here is great! I think it could've made more sense to interpolate the limit directly into the string if we were conditionally adding the `ORDER BY` and `LIMIT` clauses together (so the string would be `"ORDER BY s.provider LIMIT {}"` if we had a limit, and `""` if we didn't), but we aren't doing that—or any fancy chunking—so there's no reason to prefer it over the param.